### PR TITLE
Correct benchmark_app description of enforcebf16 parameter

### PR DESCRIPTION
**Pull request by EgorDuplensky**
### Details:
 - Just to make clear what enforcebf16 parameter is actually doing and what the options are.

### CODE REVIEW

File: inference-engine/samples/benchmark_app/benchmark_app.hpp [hunk 1]
```diff
@@ -225,7 +227,7 @@ static void showUsage() {
     std::cout << std::endl << "  device-specific performance options:" << std::endl;
     std::cout << "    -nstreams \"<integer>\"     " << infer_num_streams_message << std::endl;
     std::cout << "    -nthreads \"<integer>\"     " << infer_num_threads_message << std::endl;
-    std::cout << "    -enforcebf16              " << enforce_bf16_message << std::endl;
+    std::cout << "    -enforcebf16=<true/false>     " << enforce_bf16_message << std::endl;
```
Comment (nkogteva) [inline inference-engine/samples/benchmark_app/benchmark_app.hpp:230]: Update ReadMe as well  and make the same changes for python benchmark_app (readme and parameter description)

Comment (EgorDuplensky) [reply inference-engine/samples/benchmark_app/benchmark_app.hpp:230]: Done.
I also corrected parsing of enforcebf16 for .py version (by adding choices)

Comment (EgorDuplensky) [reply inference-engine/samples/benchmark_app/benchmark_app.hpp:230]: Hm, it seems to be an NBC change from interface perspective. I guess it is better to revert this back.

Comment (EgorDuplensky) [reply inference-engine/samples/benchmark_app/benchmark_app.hpp:230]: Actually is does not affect current interface, choices validation is performed after type conversion, so it is still possible to pass 'y' 'n' '0' '1'. Let me know if i should revert it or not.

File: inference-engine/samples/benchmark_app/README.md [hunk 2]
```diff
@@ -74,46 +74,48 @@ InferenceEngine:
...
-    -nireq "<integer>"        Optional. Number of infer requests. Default value is determined automatically for a device.
-    -b "<integer>"            Optional. Batch size value. If not specified, the batch size value is determined from Intermediate Representation.
-    -stream_output            Optional. Print progress as a plain text. When specified, an interactive progress bar is replaced with a multiline output.
-    -t                        Optional. Time, in seconds, to execute topology.
-    -progress                 Optional. Show progress bar (can affect performance measurement). Default values is "false".
-    -shape                    Optional. Set shape for input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.
-    -layout                   Optional. Prompts how network layouts should be treated by application. For example, "input1[NCHW],input2[NC]" or "[NCHW]" in case of one input size.
+    -c "<absolute_path>"        Required for GPU custom kernels. Absolute path to an .xml file with the kernels description.
+    -api "<sync/async>"         tional. Enable Sync/Async API. Default value is "async".
```
Comment (nkogteva) [inline inference-engine/samples/benchmark_app/README.md:87]: `Optional`

File: inference-engine/samples/benchmark_app/benchmark_app.hpp (hunk 1 above)
Comment (nkogteva) [reply inference-engine/samples/benchmark_app/benchmark_app.hpp:230]: It is ok for me.

File: inference-engine/samples/benchmark_app/README.md (hunk 2 above)
Comment (EgorDuplensky) [reply inference-engine/samples/benchmark_app/README.md]: Done

**Review by nkogteva — APPROVED** [review]

File: inference-engine/samples/benchmark_app/README.md [hunk 3]
```diff
@@ -74,46 +74,48 @@ InferenceEngine:
...
+    -nstreams "<integer>"       Optional. Number of streams to use for inference on the CPU, GPU or MYRIAD devices
+                                (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).
+                                Default value is determined automatically for a device.
+                                Please note that although the automatic selection usually provides a reasonable performance,
+                                it still may be non-optimal for some cases, especially for very small networks.
+                                Also, using nstreams>1 is inherently throughput-oriented option, while for the best-latency
+                                estimations the number of streams should be set to 1.
+    -nthreads "<integer>"       Optional. Number of threads to use for inference on the CPU (including HETERO and MULTI cases).
+    -enforcebf16="<true/false>" Optional. Optional. By default floating point operations execution in bfloat16 precision are enforced if supported by platform.
```
Comment (maxnick) [inline inference-engine/samples/benchmark_app/README.md:106]: Two times `Optional.` possibly a typo.

**Review by alexey-varyzgin — APPROVED** [review]

File: inference-engine/samples/benchmark_app/README.md (hunk 3 above)
Comment (EgorDuplensky) [reply inference-engine/samples/benchmark_app/README.md]: Done

**Comment by EgorDuplensky** [comment]
Rebased

**Comment by EgorDuplensky** [comment]
Rebased.
@nkogteva All comments applied. Could you please merge the PR?

**Pull request merged**